### PR TITLE
Pre-28.x branch off version bump and doc updates

### PR DESCRIPTION
**Pull request by achow101**
* Bump to 28.99 in preparation for the 28.x branching
* Remove current release note fragments. They've been moved to [draft release notes]((https://github.com/bitcoin-core/bitcoin-devwiki/wiki/28.0-Release-Notes-Draft).) in the wiki.
* Updated bips.md with missing BIPs that were implemented a while ago.

### CODE REVIEW

File: doc/release-notes-27679.md [hunk 1]
```diff
@@ -1,2 +0,0 @@
-- unix socket paths are now accepted for `-zmqpubrawblock` and `-zmqpubrawtx` with
```
Comment (tdb3) [inline doc/release-notes-27679.md:1]: nit: Adjusted this in the [Wiki Page](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/28.0-Release-Notes-Draft).  Line wrap issue.

**Review by tdb3 — APPROVED** [review]
ACK b58869f3bf5a468ef96152d357d0f8bdb7cce07d
Left nits, but resolved them in the [Wiki page](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/28.0-Release-Notes-Draft)

**Comment by luke-jr** [comment]
>Bump to 28.99 in preparation for the 28.x branching

This should be post-branching...?

**Comment by achow101** [comment]
> This should be post-branching...?

I don't think it particularly matters since that step has to be done in the 28.x branch too, in addition to the other version numbers being bumped.

**Review by glozow — COMMENTED** [review]
lgtm, should also do doc/release-notes/release-notes-27064.md

**Comment by hebasto** [comment]
> > This should be post-branching...?
> 
> I don't think it particularly matters since that step has to be done in the 28.x branch too, in addition to the other version numbers being bumped.

During the previous release cycle, it was done just _after_ the branching off.

**Review by hebasto — APPROVED** [review]
ACK d3fd608100f6f4aff2a9f7a25b43f6c5ae065024.

File: doc/bips.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ BIPs that are implemented by Bitcoin Core:
   [PR 21686](https://github.com/bitcoin/bitcoin/pull/21686)).
 * [`BIP 350`](https://github.com/bitcoin/bips/blob/master/bip-0350.mediawiki): Addresses for native v1+ segregated Witness outputs use Bech32m instead of Bech32 as of **v22.0** ([PR 20861](https://github.com/bitcoin/bitcoin/pull/20861)).
 * [`BIP 371`](https://github.com/bitcoin/bips/blob/master/bip-0371.mediawiki): Taproot fields for PSBT as of **v24.0** ([PR 22558](https://github.com/bitcoin/bitcoin/pull/22558)).
+* [`BIP 379`](https://github.com/bitcoin/bips/blob/master/bip-0379.mediawiki): Miniscript was partially implemented in **v24.0** ([PR 24148](https://github.com/bitcoin/bitcoin/pull/24148)), and fully implemented as of **v26.0** ([PR 27255](https://github.com/bitcoin/bitcoin/pull/27255)).
```
[the remark this answers is not in the document]
Comment (achow101) [reply doc/bips.md]: Fixed

**Review by tdb3 — APPROVED** [review]
re ACK 1bf9b706583572b1211762ec6ee5368bb8a2f2cd

**Pull request merged**